tests/test_browser.py

```diff
@@ -4325,6 +4325,28 @@ def test_pthread_hello_thread(self):
       for modularize in [[], ['-s', 'MODULARIZE=1', '-s', 'EXPORT_NAME=MyModule', '--shell-file', path_from_root('tests', 'shell_that_launches_modularize.html')]]:
         self.btest(path_from_root('tests', 'pthread', 'hello_thread.c'), expected='1', args=['-s', 'USE_PTHREADS=1'] + modularize + opts)
 
+  # Tests memory growth in pthreads mode, but still on the main thread.
+  @requires_threads
+  def test_pthread_growth_mainthread(self):
+    def run(emcc_args=[]):
+      # Note that this test may not pass on Chrome until 75, due to https://bugs.chromium.org/p/v8/issues/detail?id=9062
+      self.btest(path_from_root('tests', 'pthread', 'test_pthread_memory_growth_mainthread.c'), expected='1', args=['-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=2', '-s', 'ALLOW_MEMORY_GROWTH=1', '-s', 'TOTAL_MEMORY=32MB', '-s', 'WASM_MEM_MAX=256MB'] + emcc_args, also_asmjs=False)
```

Comment:
should test in PROXY_TO_PTHREAD mode too?

Reply:
that won't work yet because of [chromium issue 9065](https://bugs.chromium.org/p/v8/issues/detail?id=9065), same as `test_pthread_growth`. I'll add a `PROXY_TO_PTHREAD` mode in that test.